Project.toml

```diff
@@ -12,6 +12,7 @@ LinearAlgebra = "37e2e46d-f89d-539d-b4ee-838fcccc9c8e"
 Requires = "ae029012-a4dd-5104-9daa-d747884805df"
 Statistics = "10745b16-79ce-11e8-11f9-7d13ad32a3b2"
 Unitful = "1986cc42-f94f-5a68-af5c-568840ba703d"
+WAV = "8149f6b0-98f6-5db9-b78f-408fbbb8ef88"
```

Comment:
If `WAV` is only used for the tests then please do not include it as a dependency of the package. See e.g. https://github.com/JuliaDynamics/DrWatson.jl/blob/master/Project.toml  how the BSON dependency is tests-only.